csbindgen/src/type_meta.rs

```diff
@@ -463,20 +459,16 @@ pub fn build_method_delegate_if_required(
     method_name: &String,
     parameter_name: &String,
 ) -> Option<String> {
-    let emit_from_struct = false;
```

Comment:
This is weird. `emit_from_struct` is always false (unless I'm wrong and missing something big) which could be a choice for readability. But it's not a const, and has implications in the next line.